src/com/foamdev/demos/filter/Controller.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+// This example demonstrate the manner to filter an array of String.
+// https://docs.angularjs.org/guide/filter
+
+foam.CLASS({
+  package: 'com.foamdev.demos.filter',
+  name: 'Controller',
+  extends: 'foam.u2.Controller',// it will automatically exports: ['this as data']
+  //extends: 'foam.u2.Element',
+
+  //exports: [ 'as data' ],
+
+  css: `
+    h2 { color: #aaa; }

```

Comment:
This is global CSS, not related to this view, so it might be better to and to a <style> tag in your index.html instead.